packages/ckeditor5-engine/docs/framework/guides/deep-dive/custom-element-conversion.md

```diff
@@ -274,6 +270,75 @@ editor.conversion.for( 'dataDowncast' )
 	.add( dispatcher => dispatcher.on( 'insert:infoBox', dataDowncastConverter ) );
 ```
 
+### The model to view position mapping
+
+After adding the above downcast converters we would end up with the following view structure:
+
+```
+<div class="info-box info-box-info">
+    []<p>
+        Foobar
+    </p>
+    <div class="info-box-title">Info</div>
+    <div class="info-box-content"></div>
+</div>
+```
+
+This is not what we need. We expect the content of the `<infoBox>` to be inside the `<div class="info-box-content">`. We have the downcast conversion for the `<infoBox>` and its own structure, but we need to specify where its content should land in the view. By the default it would be converted as a direct child of `<div class="info-box">` (as you can see in the above snippet) but it should go into the `<div class="info-box-content">`. To achieve this, we need to register a callback for the {@link module:engine/conversion/mapper~Mapper#event:modelToViewPosition `Mapper#modelToViewPosition` event}, so the positions inside the model `<infoBox>` element would map to the positions inside the `<div class="info-box-content">` view element.
+
+```
```

Comment:
If you'll convert the above snippet into a table, this one could be converted too. In any case, be consistent.

Also, I don't feel `[]` helps here.

Reply:
I have no idea how to represent it as a table. I guess we could go with the proposed snippet and tune it later.